utils/validate-install-plans.js

```diff
@@ -0,0 +1,148 @@
+const path = require('path');
+const { readYamlFile, pathsToSanitize } = require('./helpers');
+const {
+  fetchPaginatedGHResults,
+  filterInstallPlans,
+} = require('./github-api-helpers');
+
+const {
+  fetchNRGraphqlResults,
+  translateMutationErrors,
+} = require('./nr-graphql-helpers');
+
+const url = pathsToSanitize[0];
```

Comment:
We ended up naming this `GITHUB_API_URL` to distinguish it from some of the other `url` variables. Might be nice to make explicit here as well

Reply:
```suggestion
const GITHUB_API_URL = passedProcessArguments[0];
```

Also renamed this array in the helpers file with 8fbf7f92

Think the sanitize naming was specific to that one file's context over here: https://github.com/tyreer/newrelic-quickstarts/blob/main/utils/sanitize_dashboards.js#L4